DEVELOPER.md

```diff
@@ -426,3 +431,56 @@ Tests should ideally follow a few conventions for easier maintenance:
   - `@pytest.mark.regression` if the test compares results from different versions
 
 **Note:** If all three external model repositories are not installed as described above, some tests will be skipped. The full test suite includes >750 cases. All must pass before changes can be merged into this repository.
+
+## Git Strategy for Managing Long-Lived Branches
+
+When a feature branch takes a long time to develop, it is easy to become out of sync with the develop branch.  Depending on the situation, it may be advisable to periodically squash the commits on the feature branch and rebase the change set with develop.  The following approach for updating a long-lived feature branch has proven robust.
+
+In the example below, the feature branch is assumed to be called `feat-xyz`
+
+### Create a Backup
+
+Begin by creating a backup copy of the feature branch in case anything goes terribly wrong.
+
+```
+git checkout feat-xyz
+git checkout feat-xyz-backup
```

Comment:
The `-b` flag is needed to create a new branch from current: `git checkout -b feat-xyz-backup`. Another functionally equivalent option is `git switch -c feat-xyz-backup`

Reply:
Good catch.  Fixed.